timely/examples/event_driven.rs

```diff
@@ -34,7 +34,7 @@ fn main() {
 
         println!("{:?}\tdataflows built ({} x {})", timer.elapsed(), dataflows, length);
 
-        for round in 0 .. {
+        for round in 0 .. 10 {
```

Comment:
We might not want to ship this change tho.

Reply:
oops.